src/plugins/settings/index.tsx

```diff
@@ -51,9 +51,11 @@ const plugin = registerPlugin<Storage>(
                 },
             } = context
 
-            for (const member of Contributors.team.concat(Contributors.contributors)) {
-                if (!member.icon) continue
+            const contributors = new Set<string>()
+            for (const member of Object.values(Contributors).flat()) {
+                if (!member.icon || contributors.has(member.name)) continue
 
+                contributors.add(member.name)
```

Comment:
Is a new set the best approach to take here? Why would there be duplicate members to begin with?

Reply:
I ran into duplicate contributors when working on Vengeance, thought I might as well commit it to rewrite as a precaution